tests/run_tests.sh

```diff
@@ -22,7 +22,7 @@ function header {
 # ui
 function ui {
 	header UI
-	python -m pytest $FLAGS $UI_URLS tests/ui/smoke_test.py tests/ui/functional_test.py
+	python -m pytest $FLAGS $UI_URLS /tests/ui/smoke_test.py tests/ui/functional_test.py
```

Comment:
Was this change intentional? I doubt it is, especially since the second path didn't have the slash added to it.

Reply:
No, it was not. We have changed it back to what it was.